src/app/components/userlist/userlist.component.ts

```diff
@@ -50,7 +50,7 @@ export class UserlistComponent {
   'clientForm': FormGroup;
   'updateForm': FormGroup;
   TicketRaised: string = '';
-  userstatus = ['In Progress', 'Pending', 'Closed', 'Improper Requirment'];
+  userstatus = ['In Progress', 'Pending', 'Closed', 'Improper Requirment' , 'Assigned'];
```

Comment:
Remove Assigned , User don't want to see assigned status, instead of that add Select Status option with empty value